#! /bin/sh -e

# DP: capi20: update to libtool-1.5

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        cd ${dir}capi20 && libtoolize --force --copy
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        cd ${dir}capi20 && libtoolize --force --copy
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
diff -ur t/isdnutils-3.3.0.20041024/capi20/configure.in isdnutils-3.3.0.20041024/capi20/configure.in
--- t/isdnutils-3.3.0.20041024/capi20/configure.in	2003-06-16 06:08:23.000000000 -0600
+++ isdnutils-3.3.0.20041024/capi20/configure.in	2004-11-27 10:20:20.189075904 -0700
@@ -1,5 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 
+AC_PREREQ(2.50)
 AC_INIT(capi20.c)
 AM_INIT_AUTOMAKE(libcapi20,2.0.6)
 AC_PREFIX_DEFAULT(/usr)
